packages/js/plugins/ethereum/src/index.ts

```diff
@@ -67,10 +67,42 @@ export class EthereumPlugin extends Module<EthereumPluginConfig> {
     _client: Client
   ): Promise<string> {
     const connection = await this._getConnection(args.connection);
-    const contract = connection.getContract(args.address, [args.method], false);
+    let method, isRawAbiMethod;
+    try {
+      method = JSON.parse(args.method);
```

Comment:
This doesn't account for the case where the caller passes in a method's ABI in JSON, but doesn't wrap in an array.

Reply:
you mean if someone would send something like.. 

`{"inputs":[],"name":"get","outputs":[{"internalType":"uint256","name":"","type":"uint256"}],"stateMutability":"view","type":"function"}`

?